src/cmake/templates/version.h.in

```diff
@@ -5,6 +5,6 @@
 #ifndef VERSION_H
 #define VERSION_H
 
-#define OSCONFIG_VERSION R""""(@OsConfigVersionString@)""""
+#define OSCONFIG_VERSION "@OsConfigVersionString@"
```

Comment:
>"@OsConfigVersionString@" [](http://example.com/codeflow?start=25&length=25)

Why doing this change?

Reply:
Not sure. If the new format builds and format correctly the version string in both gcc and clang builds, we're good. Can you please check and confirm?